packages/SketchMorph2-Core.package/M2ViewPort.class/instance/insertSelection.st

```diff
@@ -0,0 +1,7 @@
+tool selection
+insertSelection
+	super insertSelection.
+	self
+		currentCursor: ((self clipboardForm magnifyBy: self scalar) 
+			offset: (self brushSize asPoint / 2 - (self clipboardForm extent * self scalar * 0.5)));
```

Comment:
I think we should try to at least in one line use `/ 2` or `* 0.5` consistently.